### PR TITLE
5 responsive tweaks

### LINKED ISSUE #4: Assets
Ok the client did not give me an exact location of where to put said assets, so I will guesstimate where to put them and then let her tell me if she likes it there or not.

1.  This part on the home page: 
image
- Let's make this image as a background for the Heading and contact info sections. 
  - That means we need to make the image more transparent and larger height-wise.
  - That also means we might need to make the background of these said sections contrasted with black or with a background that will make it legible
  - I will not be surprised if the client does not like this, but I am trying to conserve space here as the image needs more purpose
2. On the home page, after point one, combining those parts, I want image0 and image7 to be side by side on the desktop/tablet screen and stacked on mobile. 
  - I also want to add the footer text below: "© 2024 The Royal Canine. Website created by [Kenzie Tech Solutions LLC](https://www.kenzietechsolutions.com/)
3. Place all images provided in the PhotoGallery Page.
  - The Photogallery page is not aligned with what the model page. 
  - Basically make it like https://www.shampoochgahanna.com/shampooch-gallery
    - But also keep what you have on the bottom with what you already have.

Link to assets folder: https://drive.google.com/drive/folders/1E0Zb-rXMYdFxiIeHCBvFrGyGMxBBliNW?usp=drive_link

### LINKED ISSUE #5: Responsive Tweaks
- [x] Home Screen
  - The Title and image are in a row format and must be stacked.
    - image
    - image
  - The Heading section is illegible. When completing the other ticket, the legibility should be fixed here with that, but it is still not setup for mobile. The size of the 'paragraph' should be max 300px. The title heading seems to be centered, but Business Hours and Location does not (that could do with 'business hours' being longer so maybe do a space-around so it focuses on the edges. Unsure, but play with it. Worse case scenario could be be having them stacked when it is on mobile view). 
  - On the contact info section: 
    - Capitalized contact info
- [x] Business Details & FAQ
  - image
  - There is a gap between header and 'business details'. Please make that much smaller in this mobile screen.
  - Please capitalize all headers
  - The business details section seems off-centered.
    - image
  - Based on the example website: https://www.shampoochgahanna.com/about it seems we are missing some styles. Can you please make sure you imitate the style. I know there is no disclaimers and we are adding a FAQ at the bottom, but the font seems different, horizional lines are missing, and I am unsure how you are going to add business hours location and phone information. Use fake data so we can see what it looks like. 
  - The FAQ questions in mobile form is okay until we click on the questions. That width is too small. 
    - image
    - It might be best to change them to modals when clicked to avoid this issue || The width should take a wider space on the mobile screen and not the width of the question itself.
    - Again the navbar here is not mobile friendly. 
- [x] Photo Gallery
  - The navbar (check all pages for this)
  - The modal is not mobile friendly
    - image
- [x] Events
  - Navbar only
  - Will see what client says about this.

### PR DESCRIPTION
**Pull request by Iconians**
added and updated everything in the issues plus the additional info provided via discord. I tried a React response carousel library but felt it wasn't exactly what the client would want so coded one from scratch seems to work like the provided example

### CODE REVIEW

File: src/pages/Events/Events.tsx
Comment (mikesz88) [inline src/pages/Events/Events.tsx:1]: @Iconians Where are the event examples we had here?

File: src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -67,6 +78,20 @@ export const Home = () => {
           <div />
         </div>
       </section>
+      <section className={style.imageSection}>
```
Comment (mikesz88) [inline src/pages/Home/Home.tsx:81]: Let's continue the background of what was above with the dogs.
image

Lets slide those images to the middle and make them space evenly. 
In regards to that picture of the client, we might just need to crop the image ourselves and test it out so it does not look as distorted like we see it. Make the images smaller. Let's go with something like 450 L and W and obviously responsive.

File: src/pages/Home/Home.tsx [hunk 2]
```diff
@@ -9,24 +13,32 @@ export const Home = () => {
         <div className={style.heroContainer}>
           <div className={style.heroSection}>
             <NavBar pageName={"home"} />
+            <div className={style.imgContainer}>
+              <img src={logo} alt="logo" />
+            </div>
             <div className={style.titleInfo}>
               <h1>The Royal Canine</h1>
```
Comment (mikesz88) [inline src/pages/Home/Home.tsx:20]: The name is missing part of it? 
image
image

File: src/Components/Footer/Footer.tsx [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+    <section className={styles.footerSection}>
+      <div className={styles.footer}>
+        {/* <div>
         <ul>
           <li>The Royal Canine</li>
           <li>business hours</li>
           <li>location</li>
         </ul>
+      </div> */}
+        <div className={styles.brandName}>
```
Comment (mikesz88) [inline src/Components/Footer/Footer.tsx:14]: Let's center this all the way. 
image

File: src/utils/faqArray.ts [hunk 4]
```diff
@@ -1,20 +1,61 @@
 export const faqArray = [
   {
     id: 0,
-    question: "question 1",
-    answer:
-      "Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat eligendi enim magni nesciunt atque nam nemo natus unde ea quis, quaeitaque dolores, rem totam accusantium beatae explicabo quos quod!",
+    question: "Payment at the Machine:",
```
Comment (mikesz88) [inline src/utils/faqArray.ts]: Take out the semi-colons of all the questions
image

**Review by mikesz88 — CHANGES_REQUESTED** [review]
Is this already in GitHub pages? 
How fast is the according for the images? Seems fast, but I will let the client decide. 

Make those changes and let me know when it is ready again and on GitHub pages. From there I will send it the GitHub pages to the client and get her recommended changes.

File: src/pages/Events/Events.tsx
Comment (Iconians) [reply src/pages/Events/Events.tsx:1]: under this task I thought you were to get rid of the examples, I'll put them back 
Events
Navbar only

**Comment by Iconians** [comment]
believe I fixed all those change orders

**Review by mikesz88 — APPROVED** [review]
lgtm

**Pull request merged**